### DISCUSSION TITLE
x-intersect not working on Livewire pagination

### DISCUSSION THREAD

**Discussion by kaju74** [alpinejs/alpine#2807] [7. Bugs (archived)] [answered]
I'm using the new x-intersect directive to lazy load images for the items coming from a livewire component. This items are paginated, so there's a navigation bar on the bottom to switch between the pages. The lazy loaded images workes fine on the first page but as soon as I switch to another one, the x-intersect's aren't fired, so the loading indicator is show forever. As soos as I press F5 to refresh the browser tab, the images are loaded correctly. Is this a known bug or did I had to do something else?

**Comment by SimoTod** [comment, marked as answer]
Can you post an example to reproduce?  It sounds like something specific to your implementation. Usually, elements with x-intersect work even if they are injected dynamically after the page load.

**Comment by kaju74** [comment]
Sorry for the long delay...I've already fixed it - it forgot the wire:id -> now it work's like a charme 8-)

**Reply by robdekort** [reply]
Think I'm running into the same issue, but I'm not sure what you did to solve it. If you have the time can you explain what you did?
